src/utils/utils.ts

```diff
@@ -36,8 +36,38 @@ export const imageURL = (
 
 export const getNumItems = async (uuid: string) => {
   const apiUrl = `https://api.repo.nypl.org/api/v2/collections/${uuid}/items`;
-  const apiKey = process.env.AUTH_TOKEN;
+  const res = await apiCall(apiUrl);
+  return res.numItems || 0;
+};
+
+/**
+ * Calls Repo API collectins/uuid/items endpoint
+ * @param {string} uuid - collection uuid
+ */
 
+export const getItemsFromUUID = async (uuid: string) => {
```

Comment:
This function isn't being called anywhere. Should it exist?

Reply:
I will remove it